### PR TITLE
Add changelog entry for 7487

### PR DESCRIPTION
**Pull request by dimitarvdimitrov**
follow-up of #7487

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -122,6 +122,7 @@
 * [BUGFIX] Fix metadata API using wrong JSON field names. #7475
 * [BUGFIX] Ruler: fix native histogram recording rule result corruption. #7552
 * [BUGFIX] Rules: improve error handling when querier is local to the ruler. #7567
+* [BUGFIX] Querier: fix HTTP status code translations for remote read requests. previously endpoints would return either only HTTP 400 or HTTP 500 on any error. #7487
```
Comment (duricanikolic) [inline CHANGELOG.md:125]:
```suggestion
* [BUGFIX] Querier: fix HTTP status code translations for remote read requests. Previously endpoints would return either only HTTP 400 or HTTP 500 on any error. #7487
```

Comment (duricanikolic) [reply CHANGELOG.md:125]: Should we also add something like this?

```
Currently remote-read has conflicting behaviours:
  - when returning samples all internal errors are translated to HTTP 400
  - when returning chunks all internal errors are translated to HTTP 500
```

Comment (dimitarvdimitrov) [reply CHANGELOG.md]: i included that in https://github.com/grafana/mimir/pull/7589/commits/53c4b3f8ad76564b55b8926669c5bfe1e699dddc, but tried to fit it in a single line. How does it look like?

**Review by duricanikolic — APPROVED** [review]
LGTM
Thank you @dimitarvdimitrov

**Comment by dimitarvdimitrov** [comment]
thanks for the prompt review @duricanikolic :)

**Pull request merged**